src/main/java/uk/gov/hmcts/reform/hmc/data/PartiesNotifiedEntity.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.reform.hmc.data;
```

Comment:
Not sure we need this class when we have HearingResponseEntity?

Reply:
Agreed. Will refactor